fennel/client/client.py

```diff
@@ -409,42 +243,62 @@ def extract(
                 )
         return df
 
-    def extract_features(
-        self,
-        input_feature_list: List[Union[Feature, str]],
-        output_feature_list: List[Union[Feature, Featureset, str]],
-        input_dataframe: pd.DataFrame,
-        log: bool = False,
-        workflow: Optional[str] = None,
-        sampling_rate: Optional[float] = None,
-    ) -> Union[pd.DataFrame, pd.Series]:
+    # ----------------------- Branch API's -----------------------------------
+
+    def empty_branch(self, name: str):
```

Comment:
All other API methods start with a verb except this one. Consider using `create_branch` or `init_branch`